pkg/headers/iphlpapi/types.go

```diff
@@ -0,0 +1,11 @@
+package iphlpapi
+
+// MIB_TCPROW_OWNER_PID represents a row in the TCP connection table
```

Comment:
I have the feeling that MIB_TCPROW_OWNER_PID just works, if af =IPv4.

If af is IPv6, the struct seems different. https://learn.microsoft.com/de-de/windows/win32/api/tcpmib/ns-tcpmib-mib_tcp6row_owner_pid

For example, dwLocalAddr is an [16]byte (128bit) value, which is 4 times larger than uint32. This could be the reasons, why IPv6 has some unknown values.

```
windows_tcp_connection_state_count{af="ipv6",state="UNKNOWN_0"} 16
```

Reply:
now it works correctly